src/main/resources/ValidationMessages.properties

```diff
@@ -9,8 +9,9 @@ GuestTeacher.locale.required=The locale of the guest teacher is required.
 School.name.required=The name of the school is required.
 School.address.required=The address of the school is required.
 School.postalCode.required=The postal code of the school is required.
-School.contactPersonName.required=The name of the contact person is required.
 School.contactPersonEmail.invalid=The email address of the contact person is invalid.
 School.contactPersonEmail.required=The email address of the contact person is required.
```

Comment:
Should the School.contactPersonEmail, GuestTeacher.email & GuestTeacher.locale be removed? I believe they are not used anymore.

Reply:
You can check if this is the case. But I think you are right. You can remove those.